spec.html

```diff
@@ -40244,17 +40215,102 @@ <h1>AsyncGeneratorYield ( _value_ )</h1>
           1. Let _generator_ be the value of the Generator component of _genContext_.
           1. Assert: GetGeneratorKind() is ~async~.
           1. Set _value_ to ? Await(_value_).
-          1. Set _generator_.[[AsyncGeneratorState]] to ~suspendedYield~.
-          1. Remove _genContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
-          1. Set the code evaluation state of _genContext_ such that when evaluation is resumed with a Completion _resumptionValue_ the following steps will be performed:
-            1. If _resumptionValue_.[[Type]] is not ~return~, return Completion(_resumptionValue_).
-            1. Let _awaited_ be Await(_resumptionValue_.[[Value]]).
-            1. If _awaited_.[[Type]] is ~throw~, return Completion(_awaited_).
-            1. Assert: _awaited_.[[Type]] is ~normal~.
-            1. Return Completion { [[Type]]: ~return~, [[Value]]: _awaited_.[[Value]], [[Target]]: ~empty~ }.
-            1. NOTE: When one of the above steps returns, it returns to the evaluation of the |YieldExpression| production that originally called this abstract operation.
-          1. Return ! AsyncGeneratorResolve(_generator_, _value_, *false*).
-          1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
+          1. Let _completion_ be NormalCompletion(_value_).
+          1. Assert: There are at least two items on the execution context stack.
+          1. Let _previousContext_ be the execution context immediately beneath the running execution context.
```

Comment:
```suggestion
          1. Assert: The execution context stack has at least two elements.
          1. Let _previousContext_ be the second to top element of the execution context stack.
```
(precedent in [19.2.1 eval(x)](https://tc39.es/ecma262/#sec-eval-x) and [20.2.1.1.1 CreateDynamicFunction](https://tc39.es/ecma262/#sec-createdynamicfunction))